provisioning/azure-devops-deploy.sh

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env bash
 set -Eeuo pipefail
 
-COMMIT=$(echo "$BUILD_BUILDID" | cut -d'-' -f 2)
-export COMMIT
-
 # Checkout correct commit
 if [ "$RELEASE_ARTIFACTS__KEBOOLA_KEBOOLA_AS_CODE_SOURCEBRANCH" == "master" ]; then
+  COMMIT=$(echo "$BUILD_BUILDID" | cut -d'-' -f 2)
```

Comment:
Ještě, že tam je ten `master`, protože by to nikdy stejně nefungovalo (nevyparsuje to nic, co by šlo checkoutnout) :-) Podle mě to můžeš celý smazat a můžem se spíš bavit o tom, jak moc citlivý je release appky na to, když je image a release pipelina puštěný z jinýho commitu. 

Pokud to není úplně jasný, můžem si nad tím zavolat. Je to tady dost možná zbytečná obstrukce.

Reply:
No jo vlastně. 😄 

No, já si nedovedu úplně představit kdy může taková situace nastat, že se do artefaktů pipeliny dostane image a repo na rozdílných commitech. (Pokud to samozřejmě nenastavím manuálně.)

Reply:
Jednoduše, stačí, když mezi spuštěním build pipeliny (s tagem) a releasem image (s tagem) bude v main branchi novej commit.

Reply:
Jo. 🤦‍♂️ To se zatím moc neděje, ale asi by nebylo špatný to fixnout teď ať se v budoucnu nedivíme. 🙂

Reply:
Nikde se nám to zatím nestalo, takže to klidně smaž, jen stačí vědět, že se tohle může stát.

Reply:
ok: https://github.com/keboola/keboola-as-code/pull/758